leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/LeshanServer.java

```diff
@@ -131,7 +131,8 @@ public void registered(final Client client) {
 
         // default endpoint
         coapServer = new CoapServer();
-        nonSecureEndpoint = new CoapEndpoint(localAddress);
+        nonSecureEndpoint = new CoapEndpoint(localAddress, NetworkConfig.getStandard(), this.observationRegistry,
+                this.observationRegistry.getObserveRequestStore());
```

Comment:
I'm trying to test this pull request, but in the specified californium PR #16 i cannot find the constructor with four arguments in class CoapEndpoint. I'm missing the CaliforniumObservationRegistry. Thereby getting an error here in LeshanServer.